bom/compile/pom.xml

```diff
@@ -238,13 +238,14 @@
     <dependency>
       <groupId>io.swagger</groupId>
       <artifactId>swagger-annotations</artifactId>
-      <version>1.5.5</version>
+      <version>1.6.1</version>
```

Comment:
Do you know if it also compiles with the older version? If that's the case it's better to use older versions here. That way a broader version range is generated by Bnd and the framework will be more compatible with older Swagger versions. That way you don't force framework users to upgrade dependencies or use multiple versions, i.e. it could result in similar issues as we often have with guava and the org.reflections artifact.

Reply:
Ah, thanks, it compiled fine before, the version change was done at the last minute (what I thought was) some regular housekeeping, I'll change it back then.